xbmc/addons/kodi-addon-dev-kit/include/kodi/xbmc_pvr_types.h

```diff
@@ -237,6 +237,7 @@ extern "C" {
   {
     const char* strUserPath;           /*!< @brief path to the user profile */
     const char* strClientPath;         /*!< @brief path to this add-on */
+    int iEpgMaxDays;                   /*!< @brief if > EPG_TIMEFRAME_UNLIMITED, in async epg mode, deliver only events in the range from 'end time > now' to 'start time < now + iEpgMaxDays. EPG_TIMEFRAME_UNLIMITED, notify all events. */
```

Comment:
Can you use this way?
```cpp

    /// if > EPG_TIMEFRAME_UNLIMITED, in async epg mode, deliver only events in
    /// the range from 'end time > now' to 'start time < now + iEpgMaxDays.
    /// EPG_TIMEFRAME_UNLIMITED, notify all events.
    int iEpgMaxDays;
```
Is for two reasons, the first is to use the soon comming new add-on documentation and the other, `@brief` gives problems on doxygen related to structure values and text not used with them.